config/loggers.js

```diff
@@ -0,0 +1,56 @@
+require('dotenv').config();
+const winston = require('winston');
+const { format, transports } = winston;
+const { combine, label, colorize, printf, timestamp } = format;
+
+const logFormat = printf(({ level, message, label = '', timestamp }) => {
+  return `${label}[${timestamp}] ${level}: ${message}`;
+});
+
+/**
+ * Modifies winston global config to add two loggers.
+ * 
+ * @returns {void}
+ */
+module.exports.configureAppLogging = () => {
+    /**
+     * Adds a default logger.
+     */
+    winston.loggers.add('defaultLog', {
+        silent: 'true' === process.env.SILENCE_DEFAULT_LOG || false,
+        transports: [
+            new transports.Console({
+            level: 'info',
```

Comment:
this part here might benefit from having an env variable 
```suggestion
            level: process.env.LOGGING_LEVEL || 'info',
```